README.md

```diff
@@ -51,7 +51,7 @@ Next, lets configure the development server. To keep things simple we can put th
 var hypernova = require('hypernova/server');
 
 hypernova({
-  devMode: true,
+  clustering: false,
```

Comment:
Additionally, it might be nice to do the option rename in a separate PR. Not a huge deal tho.